ipie/qmc/afqmc.py

```diff
@@ -288,7 +289,10 @@ def copy_to_gpu(self):
                     )
             self.propagator.cast_to_cupy(self.verbose and comm.rank == 0)
             self.hamiltonian.cast_to_cupy(self.verbose and comm.rank == 0)
-            self.trial.cast_to_cupy(self.verbose and comm.rank == 0)
+            if type(self.trial)==ParticleHole:
```

Comment:
if not isinstance(self.trail, ParticleHole):
    self.trial.cast_to_cupy(self.verbose and comm.rank == 0)